email/email.go

```
@@ -0,0 +1,34 @@
+package email
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/Clever/http-science/config"
+	"github.com/Clever/http-science/science"
+	"github.com/keighl/mandrill"
+)
+
+// SendEmail sends email to the address in the payload with the results
+func SendEmail(payload *config.Payload, duration time.Duration, res science.Results) error {
+	message := &mandrill.Message{}
+	message.AddRecipient(payload.Email, "", "to")
+
+	science.Res.Mutex.Lock()
+	message.GlobalMergeVars = mandrill.MapToVars(map[string]interface{}{
+		"TYPE":       payload.JobType,
+		"REQS":       strconv.Itoa(res.Reqs),
+		"NUM_DIFFS":  strconv.Itoa(res.Diffs),
+		"DIFFS_MAP":  fmt.Sprintf("%#v", res.Codes),
+		"DIFFS_FILE": payload.DiffLoc,
+		"TIME":       duration.String(),
+	})
+	science.Res.Mutex.Unlock()
+	templateName := "science-correctness"
+
+	mandrillClient := mandrill.ClientWithKey(os.Getenv("MANDRILL_KEY"))
```

nit: try to move any configuration setup to the first few things that happen in main, so the app will fail hard immediately if env is not configured